src/traffic_server/InkAPI.cc

```diff
@@ -1454,6 +1454,7 @@ HttpHookState::Scope::operator++()
 void
 HttpHookState::Scope::clear()
 {
+  _hooks = nullptr;
```

Comment:
This change is needed with or without the changes to the class allocators.  I think the only reasons we are not seeing crashes in master currently is that, by luck, the prototype object has zeros in this field.

Reply:
@duke8253

Reply:
The change to this file is now a separate PR:  https://github.com/apache/trafficserver/pull/6275